.gitignore

```diff
@@ -138,3 +138,6 @@ tests/**/runs/**
 tests/**/tmp*/**
 open_model_zoo/
 nncf-tests.xml
+env-migration/
+tests/post_training/data/
```

Comment:
Why did you add `env-migration/` and `tests/post_training/data/` to the gitignore?

Reply:
This is from my local implementation, I will remove it now.

Reply:
Done